### PR TITLE
recommend quickplay prompts

### LINKED ISSUE #444: Component that suggests quick play prompts from other users
Rather than having users rely on their own ideas or our generator to populate the quick play component, we could also have a list of prompts recently played (perhaps excluding randomly generated ones).

### PR DESCRIPTION
**Pull request by mliu59**
Adding a "recommended prompts" section to the quick play panel. On default loads the most recent 5 FINISHED quick runs and shows a play button for players to quickly jump into. 

image

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
     "SECRET_KEY": "ahhhhhhhhhh",
 
     "MYSQL_HOST": "127.0.0.1",
-    "MYSQL_USER": "user",
-    "MYSQL_PASSWORD": null,
+    "MYSQL_USER": "root",
```
Comment (dqian3) [inline config/default.json:5]: oops, put your local stuff in `config/prod.json`

**Review by dqian3 — CHANGES_REQUESTED** [review]
Small change. Also I would say that these shouldn't be hidden completely in a `<details>` element. Maybe show 1 or 2 and an option to see more. Also would be nice to see the time it was played, i.e. using `moment.js`

**Comment by bricehalder** [comment]
+ this would be an easy target to grief i.e. putting nsfw stuff in the quick plays box

**Comment by mliu59** [comment]
> plus this would be an easy target to grief i.e. putting nsfw stuff in the quick plays box.
> 
> Maybe instead of most recent, we could do something like most played in the past day or something?

I changed the suggested prompts to be random finished prompts from the past 3 days, so griefing would be practically unviable with sufficient traffic volume. I think griefing would be easier to do if it was just most played since they can just spam.

**Review by dqian3 — APPROVED** [review]

**Pull request merged**